scripts/release/build-release.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+set -ex
+set -o pipefail
+
+# check config
+# BUILD_COMMIT
+# NEXUS_USER
+# NEXUS_PW
+# GPG_SIGNING_KEY
+
+# checkout
+git checkout $BUILD_COMMIT
+
+# generate source artifacts
+sbt sourceDistGenerate
+
+# sign source artifacts
+find target/dist -regex '.*\(tgz\|zip\)' | xargs -I{} sh -c "sha512sum {} > {}.sha512"
```

Comment:
newer versions (that already exist, but not yet uptaken here) produce the sha digests automatically

Reply:
I will make a PR to update version now.

Reply:
So the PR has been merged (see https://github.com/apache/incubator-pekko/pull/195) so you don't need to worry about manually creating digests of the files.